src/platform/macos.rs

```diff
@@ -38,6 +38,12 @@ pub trait WindowExtMacOS {
 
     /// Sets whether or not the window has shadow.
     fn set_has_shadow(&self, has_shadow: bool);
+
+    /// Get window z-level
+    fn get_window_level(&self) -> i64;
+
+    /// Set window z-level
+    fn set_window_level(&self, level: i64);
```

Comment:
These should return/take an `isize` instead of an `i64`, because an `NSInteger` is 32 bits wide when compiled to a 32 bit architecture.

Furthermore I think these should just be called `get_level` and `set_level`; I don't think that adding "window" provides any information here. I think the documentation could include a link to the appropriate apple developers link, like
https://developer.apple.com/documentation/appkit/nswindow/1419511-level?language=objc